examples/interactions/injections.py

```diff
@@ -12,9 +12,8 @@
 
 bot = commands.Bot(command_prefix=commands.when_mentioned)
 
-# Instead of repeating boiler-plate code you may use injections
```

Comment:
This example throws an error when attempting to run it.

Reply:
#770